src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -8,7 +8,23 @@ public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws InvalidDataException {
+        if (user.getAge() == null || user.getLogin() == null || user.getPassword() == null) {
+            throw new InvalidDataException("All required fields should be completed ");
```

Comment:
Instead of checking all fields in one condition, consider throwing specific `InvalidDataException` messages for each null field. This will provide clearer feedback to the user about which field is missing.